site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -133,12 +132,12 @@ const DisruptionDetail = ({ disruption, redirectCookie, csrfToken }: DisruptionD
         <BaseLayout title={title} description={description}>
             {popUpState && csrfToken ? (
                 <DeleteConfirmationPopup
-                    entityName={"the disruption"}
-                    deleteUrl={"/api/delete-disruption"}
+                    entityName={`the ${popUpState.name}`}
```

Comment:
Could the entire text value `Are you sure you wish to delete the $${popUpState.name}` be passed? This way the text displayed in the component can be customised